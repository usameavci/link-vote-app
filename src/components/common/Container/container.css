.common-container {
  padding: 1rem;
  margin: 0 auto;
  max-width: 460px;
}
