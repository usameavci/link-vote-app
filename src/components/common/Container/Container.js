import React from "react";

import "./container.css";

const Container = ({ children }) => {
  return <div data-testid="common-container" className="common-container">{children}</div>;
};

export default Container;
