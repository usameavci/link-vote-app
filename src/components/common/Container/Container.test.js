import React from "react";
import renderer from "react-test-renderer";
import { render } from "@testing-library/react";

import Container from "./Container";

describe("Container component tests", () => {
  it("renders without crashing", async () => {
    const component = <Container />;

    const { getByTestId } = render(component);

    expect(getByTestId("common-container")).toBeInTheDocument();
  });

  it("matches snapshots", () => {
    const component = <Container />;
    const tree = renderer.create(component);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
