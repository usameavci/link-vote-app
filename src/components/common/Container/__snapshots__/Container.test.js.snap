// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container component tests matches snapshots 1`] = `
<div
  className="common-container"
  data-testid="common-container"
/>
`;
